src/styles/components/pagination.scss

```diff
@@ -38,6 +38,12 @@ $active-link-bg-color: var(--active-link-bg-color);
                     color: $link-hover-txt-color;
                 }
             }
+            &:focus {
+                box-shadow: none;
+            }
+            &:active{
+                box-shadow: inset 0 0 0 .2rem rgba(0,123,255,.25);
```

Comment:
@petrKavulok Are you sure about that rgba color? Why dont you use our `$primary` color ?

Reply:
Also, will your solution work properly in dark mode?